src/CKAN.jl

```diff
@@ -0,0 +1,60 @@
+struct CKAN <: DataRepo
+end
+
+base_url(::CKAN) = "https://demo.ckan.org/api/3/action/package_show?id="
```

Comment:
I don't think this is useful, given there are many sites that use this,
There is no single canonical `base_url`.